### DISCUSSION TITLE
harmonyos/development-environment-dajian

### DISCUSSION THREAD

**Discussion by giscus** [vuepress-theme-hope/giscus-discussions#872] [Announcements]
# harmonyos/development-environment-dajian

开发鸿蒙的应用需要借助 HUAWEI DevEco Studio 1.3.1. 下载安装包 https://developer.harmonyos.com/cn/develop/deveco-studio/

http://localhost:8080/harmonyos/development-environment-dajian.html

**Comment by yuanjing001** [comment]
你好
